apps/preview/nuxt/pages/showcases/RatingForms/ProductRating.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <SfButton type="button" class="absolute right-1/2 top-1/2 translate-x-[50%]" @click="open"
+    >Open rating modal again</SfButton
+  >
+  <SfModal
+    v-model="isOpen"
+    class="min-w-[376px] md:w-[480px]"
+    tag="section"
+    role="alertdialog"
+    aria-labelledby="modalTitle"
+    aria-describedby="modalDesc"
+  >
+    <header>
+      <SfButton
+        square
+        variant="tertiary"
+        aria-label="Close product rating modal"
+        size="sm"
+        class="!text-neutral-500 absolute right-2 top-2"
+        @click="close"
+      >
+        <SfIconClose size="sm" />
+      </SfButton>
+      <h3 id="modalTitle" class="font-bold typography-headline-4 md:typography-headline-3 mb-8">Rate your purchase</h3>
```

Comment:
I think we should use `useId` here as well. The only concern I have is that in case of React the hook is built-in the framework, but in Vue it's our custom composable. Nevertheless I think it's better to have IDs generated and uniq.

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2849/commits/d3797690dbd108bf5bf4029e2c4270327ee10e2a